src/Model/Creditor.php

```diff
@@ -55,11 +55,11 @@ class Creditor extends Model
     protected $eur_payout_account;
 
     /**
-     * @param string $address_line1
+     * @param string $addressLine1
      */
-    public function setAddressLine1($address_line1)
+    public function setAddressLine1($addressLine1)
     {
-        $this->address_line1 = $address_line1;
+        $this->address_line1 = $addressLine1;
```

Comment:
@martin-georgiev what is the reason we are changing to camelcase for arguments and still have snake case for class vars? Is this in the pipeline?

Reply:
@alexlongshaw The reason for arguments' case-change is to unify the look and feel of method signatures across the project. Models are the odd ones, as they make use of `toArray()` to represent [GoCardless API objects](https://developer.gocardless.com/api-reference/#core-endpoints-creditors). Those API objects define snake-case properties. As long as we use `toArray()`, our model class properties will have to stay snake-case. I plan to address this in a future PR.